MSAL/src/public/MSALDefinitions.h

```diff
@@ -178,6 +178,29 @@ typedef NS_ENUM(NSUInteger, MSALPlatformSSOStatus)
     MSALPlatformSSOEnabledAndRegistered
 };
 
+/**
+ Device mode configured by the administrator
+ */
+typedef NS_ENUM(NSUInteger, MSALQRPinAvailability)
```

Comment:
See my feedback in common core - should we make this a generic preferred_auth_method configuration rather than something specific to QR+PIN?